File: iniciante/1038/1038.cpp
// 1038 - Lanche - beecrowd
// Iniciante - Nível 1

#include <iostream>
#include <iomanip>
#include <vector>

int main()
{
    const std::vector<std::pair<int, double>> CARDAPIO = {
        {1, 4.00}, 
        {2, 4.50},
        {3, 5.00},
        {4, 2.00},
        {5, 1.50}
    };

    int codigo, quantidade;
    std::cin >> codigo >> quantidade;

    double total = 0.0;

    for(auto pedido: CARDAPIO)
    {
        if(pedido.first == codigo)
        {
            total = pedido.second * quantidade;

            std::cout << std::fixed << std::setprecision(2)
                      << "Total: R$ " << total << '\n';
            break;
        }
    }

    return 0;
}

File: iniciante/1038/1038.py
# 1038 - Lanche - beecrowd
# Iniciante - Nível 1

CARDAPIO = {
    1: 4.00,
    2: 4.50,
    3: 5.00,
    4: 2.00,
    5: 1.50
}

codigo, quantidade = input().split()
codigo = int(codigo)
quantidade = float(quantidade)

total = CARDAPIO[codigo] * quantidade

print(f"Total: R$ {total:.2f}")
